bindata/network/kuryr/003-credentials.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kuryr-config-credentials
+  namespace: openshift-kuryr
+stringData:
+  kuryr-credentials.conf: |+
+    {{- $AuthInfo := .OpenStackCloud.AuthInfo }}
+    [neutron]
+    auth_type = {{ default "password" .OpenStackCloud.AuthType }}
+    auth_url = {{ $AuthInfo.AuthURL }}
+    insecure = {{ .OpenStackInsecureAPI }}
+    token = {{ default "\"\"" $AuthInfo.Token }}
```

Comment:
we need to keep iniEscapeCharacters otherwise it will fail if $ is present

Reply:
Yes, thank you!